test/functionalTest/cases/117_convop_using_standard_ops/postSubscribeContextAvailabilityConvOp.test

```diff
@@ -0,0 +1,226 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+ConvOp postSubscribeContextAvailabilityConvOp: POST /v1/registry/contextAvailabilitySubscriptions
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0
+accumulatorStart
+
+--SHELL--
+
+#
+# 01. Subscribe to E1/T1 (POST /v1/registry/contextAvailabilitySubscriptions)
```

Comment:
BTW: Probably not.

I just looked up some info on notifyFormat and it is only looked at in mongoBackend/mongoSubscribeContext.cpp ...
So, it will not work for this test, unless I modify mongoSubscribeContextAvailability.